admin/metrics/views.py

```diff
@@ -0,0 +1,26 @@
+from django.views.generic import ListView
+from django.shortcuts import redirect
+from django.core.urlresolvers import reverse
+from djqscsv import render_to_csv_response
+
+from .models import OSFStatistic
+from .utils import get_osf_statistics
+
+
+def update_metrics(request):
+    get_osf_statistics()
+    return redirect(reverse('metrics:stats_list'))
+
+
+def download_csv(request):
+    queryset = OSFStatistic.objects.all()
```

Comment:
Date descending would be fine, to match [the `ListView`](https://github.com/CenterForOpenScience/osf.io/blob/349e0e061372413688133b48562799addb6524b6/admin/metrics/views.py#L26) just below.